metadata-models/src/main/pegasus/com/linkedin/dataprocess/DataProcessInstanceProperties.pdl

```diff
@@ -0,0 +1,110 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.CustomProperties
+import com.linkedin.common.ExternalReference
+import com.linkedin.common.Urn
+import com.linkedin.common.JobFlowUrn
+import com.linkedin.common.DataJobUrn
+
+/**
+ * The inputs and outputs of this data process
+ */
+@Aspect = {
+  "name": "dataProcessInstanceProperties",
+  "type": "timeseries",
+}
+record DataProcessInstanceProperties includes CustomProperties, ExternalReference {
+
+  /**
+   * Process name
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
+
+  /**
+   * Audit stamp containing who reported the lineage and when
+   */
+  auditStamp: AuditStamp
+  
+  /**
+   * Process description
+   */
+  @Searchable = {
+    "fieldType": "TEXT",
+    "hasValuesFieldName": "hasDescription"
+  }
+  description: optional string
+
+  /**
+   * Process type
+   */
+  @Searchable = {
+    "fieldType": "KEYWORD",
+    "addToFilters": true,
+    "filterNameOverride": "Process Type"
+  }
+  type: optional enum DataProcessType {
```

Comment:
Let's minimize the number of types we add here in the beginning. 

Maybe just BATCH_SCHEDULED, BATCH_AD_HOC?

Reply:
The main reason when I raised this request https://feature-requests.datahubproject.io/b/Developer-Experience/p/entity-model-for-streaming-applications-or-microservices was for modelling kafka stream applications. I added the microservices requirement just for completion of the request.

So kindly requesting the `STREAMING` value here 😄

Reply:
I added it back

Reply:
Got it- thanks @sgomezvillamor !